packages/preset-mini/src/_rules/border.ts

```diff
@@ -80,6 +80,12 @@ function borderColorResolver(direction: string) {
       }
     }
     else if (color) {
+      if (color.startsWith('calc')) {
+        return {
+          'border-width': color,
+        }
+      }
```

Comment:
This does not sound like a long term solution. Do you have reference in TW's codebase?

Reply:
There are other utilities with the same issue like `outline` so you would have to go utility by utility this way.

I looked at the TW code and it seems like they have a regex checking for `min`, `max`, `clamp` or `calc`. Not sure if it could work here too
reference: https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/dataTypes.js#L9

Reply:
> There are other utilities with the same issue like `outline` so you would have to go utility by utility this way. [There are other utilities with the same issue like outline so you would have to go utility by utility this way.]
> 
> I looked at the TW code and it seems like they have a regex checking for `min`, `max`, `clamp` or `calc`. [I looked at the TW code and it seems like they have a regex checking for min, max, clamp or calc.] Not sure if it could work here too [Not sure if it could work here too] reference: [https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/dataTypes.js#L9 [reference: https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/dataTypes.js#L9] ](https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/dataTypes.js#L9)

thx, I add `isCSSFunction` to replace `color.startsWith('calc')`